Open Judge System/Data/OJS.Data.Models/Problem.cs

```diff
@@ -44,11 +44,6 @@ public class Problem : DeletableEntity, IOrderable
 
         public int OrderBy { get; set; }
 
-        /// <summary>
-        /// Gets or sets a predefined skeleton for the task
-        /// </summary>
-        public byte[] SolutionSkeleton { get; set; }
```

Comment:
If I remove the property it will be removed from the DB as well and the data for 'SolutionSkeleton' from table Problems should be deleted too.
If we don't remove it it will stay as a backup and persist in the DB.
Should I remove this? Any thoughts @Minkov @shsimeonova on this?

IMO
Removing it and deleting data after it sounds fine to me